todo/src/terminal/error.rs

```diff
@@ -1,20 +1,23 @@
-use std::{io::Error};
+use std::{
+    fmt::{self},
```

Comment:
```suggestion
    fmt,
```

Eu acho que o rustfmt iria ter arrumado isso.